ui/widgets/nuxeo-user-avatar.js

```diff
@@ -299,33 +307,40 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
           let tempName = '';
           const splitName = name.split(' ');
           const alphabetPosition = alphabet.indexOf(name.charAt(0).toLowerCase());
-          this.$.container.style.backgroundColor = colors[alphabetPosition];
-          for (let i = 0; i < splitName.length; i++) {
-            tempName += splitName[i].charAt(0);
-          }
-          this._output = tempName;
-          if (this.fetchAvatar) {
-            this.$.getUserProfile.path = `user/${this._username(this.user)}`;
-            this.$.getUserProfile
-              .get()
-              .then((res) => {
-                if (
-                  res.contextParameters &&
-                  res.contextParameters.userprofile &&
-                  res.contextParameters.userprofile.avatar
-                ) {
-                  this.user = res;
-                }
-              })
-              .catch(() => {
-                console.warn(`Cannot fetch profile for user ${this._username(this.user)}`);
-              });
+          if (alphabetPosition === -1) {
+            // ELEMENTS-903: If case user's data contains non Latin characters
+            this._isInTheAlphabet = false;
+            const randomColor = colors[Math.floor(Math.random() * colors.length)];
+            this.$.container.style.backgroundColor = randomColor;
+          } else {
+            this._isInTheAlphabet = true;
+            this.$.container.style.backgroundColor = colors[alphabetPosition];
+            for (let i = 0; i < splitName.length; i++) {
+              tempName += splitName[i].charAt(0);
+            }
+            this._output = tempName;
+            if (this.fetchAvatar) {
```

Comment:
Is this fetchAvatar logic specific for the "latin chars" scenario? Shouldn't it be used in both cases?